code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -265,11 +265,25 @@
 	R.set_frequency(radio_freq)
 	synd_mob.equip_to_slot_or_del(R, slot_l_ear)
 
+	var/back = /obj/item/storage/backpack
+
+	switch(synd_mob.backbag)
+		if(GBACKPACK, DBACKPACK)
```

Comment:
You can probably just remove this check since the else case handles backpacks anyway

Reply:
Don't particularly like relying on the default case if possible but can be changed if you want me to.